.github/CODEOWNERS

```diff
@@ -0,0 +1,14 @@
+# For more information on how CODEOWNERS work, see
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners.
+
+# Default owner for everything in the repo. Matches farther down in the file
+# will override this default.
+* @urbit/runtime
```

Comment:
github is saying this "contains errors" about this line
<img width="777" alt="Screenshot 2023-01-19 at 5 35 39 PM" src="https://user-images.githubusercontent.com/441466/213599243-60168e12-9e34-486e-92a5-447110cae32c.png">

Reply:
I hadn't explicitly added `@urbit/runtime` as a team to this repo. I just did that, and it should be good now.